src/main/java/DBConnector.java

```diff
@@ -0,0 +1,179 @@
+import org.sqlite.*;
+
+import java.sql.*;
+
+public class DBConnector {
+
+    static String dbName;
+    static String url;
+    static String cardTableName;
+    static SQLiteDataSource dataSource = new SQLiteDataSource();
+
+    public static void createDatabase(String dbname) {
+        dbName = dbname;
+        url = "jdbc:sqlite:" + dbName;
+        dataSource.setUrl(url);
+        try (Connection con = dataSource.getConnection()) {
+            try (Statement statement = con.createStatement()) {
+            } catch (SQLException e) {
+                e.printStackTrace();
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void createTable(String name) {
+        cardTableName = name;
+        try (Connection conn = DriverManager.getConnection(url);
+             Statement stmt = conn.createStatement()) {
+            stmt.executeUpdate("CREATE TABLE IF NOT EXISTS card ("
+                    + "id INTEGER PRIMARY KEY,"
+                    + "number text NOT NULL,pin TEXT NOT NULL,"
+                    + "balance INTEGER DEFAULT 0)");
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public static void insertCard(Account account) {
+        String sql = "INSERT INTO " + cardTableName + " VALUES "
+                + "	(" + account.getId() + "," + account.getCardNumber() + "," + account.getPIN() + "," + account.getBalance() + ");";
+        try (Connection conn = DriverManager.getConnection(url);
+             Statement stmt = conn.createStatement()) {
+            int i = stmt.executeUpdate(sql);
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+    }
+
+    public static int getCount() {
+        int id = 0;
+        try (Connection conn = DriverManager.getConnection(url);
+             Statement stmt = conn.createStatement()) {
+
+            ResultSet res = stmt.executeQuery("Select * from " + cardTableName + ";");
+            while (res.next()) {
+                id = res.getInt("id");
+            }
+
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+        return id;
+    }
+
+    public static boolean containCard(String cardNumber) {
+        String contains = "SELECT count(number) FROM " + cardTableName + " WHERE number = '" + cardNumber + "';";
```

Comment:
Can you use predefined queries wherever possible? Please read about `prepareStatement `.

Reply:
fixed